Paper.tex

```diff
@@ -66,6 +66,7 @@
     Dr. Gavin Wood\\
     Founder, Ethereum \& Parity\\
     gavin@parity.io
+    And others as per https://github.com/ethereum/yellowpaper/commits/master
```

Comment:
Please change this to `And others as detailed \href{https://github.com/ethereum/yellowpaper/graphs/contributors}{here}.`